app/README.md

```diff
@@ -1,70 +1,73 @@
-# Getting Started with Create React App
+# Frontend for Learning Journey Planning System
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This is the application root for the frontend system of LJPS.
 
-## Available Scripts
+## Prerequisites
 
-In the project directory, you can run:
+- Recommended platform
+    - Development: macOS Monterey (Intel)
+    - Production: Linux (Ubuntu 16.04)
+- Nodejs 14.x, 16.x, 18.x (v18.9.1 preferred)
+    - Accompanying npm (v8.19.2 preferred)
+- Running backend for data fetching **[STRONGLY RECOMMENDED]**
+    - This is crucial to load any meaningful logic/data, without the backend, the application will appear as an empty shell
 
-### `yarn start`
+## Installation and Set-up
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
-
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
-
-### `yarn test`
-
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `yarn build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+This application can be run in both `development` and `production` modes. However, they share the following set-up instructions.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### Set-up Local Directories
 
-### `yarn eject`
+Clone this repository or download the files to local directory.
+Open a terminal session and navigate to this application root (`.../is212g5t7-project/app`)
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+```bash
+cd /path/to/is212g5t7-project/app
+```
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### Provide Environment Variables
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+This frontend application relies on the accompanying backend server for computation logic and data. We need to provide it the following information. Edit the backend server URL in `.env.example` using any text editor (`vi .env.example`).
+    1. Replace `<>` fields with the respective information (backend server URL)
+    2. Rename `.env.example` to `.env`
```

Comment:
Not sure if the screenshot has been updated, but I believe this has the same issue where the ordered list not displaying as intended.